LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
should be: https://github.com/bitrise-steplib/step-template/blob/master/LICENSE

! `Krisztian Dobmayer`